addOns/help/src/main/javahelp/contents/releases/2.12.0.html

```diff
@@ -184,8 +184,12 @@ <H2>ZAP API Deprecated Endpoints:</H2>
 <li>ACTION core / setOptionUseSocksProxy</li>
 <li>ACTION localProxies / addAdditionalProxy</li>
 <li>ACTION localProxies / removeAdditionalProxy</li>
+<li>OTHER core / messageHar</li>
+<li>OTHER core / messagesHar</li>
+<li>OTHER core / messagesHarById</li>
 <li>OTHER core / proxy.pac</li>
 <li>OTHER core / rootCaCert</li>
+<li>OTHER core / sendHarRequest</li>
```

Comment:
They need to be moved to other listing, these are the ones superseded by the network add-on.

Reply:
Oops, okay